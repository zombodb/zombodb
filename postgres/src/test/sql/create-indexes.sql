set datestyle to 'iso, mdy';

CREATE INDEX idxso_posts ON so_posts USING zombodb (zdb(so_posts)) WITH (url='http://localhost:9200/');
CREATE INDEX idxso_users ON so_users USING zombodb (zdb(so_users)) WITH (url='http://localhost:9200/');
CREATE INDEX idxwords ON words USING zombodb (zdb(words)) WITH (url='http://localhost:9200/');
