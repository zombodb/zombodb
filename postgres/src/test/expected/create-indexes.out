CREATE INDEX idxso_posts ON so_posts USING zombodb (zdb('so_posts', ctid), zdb(so_posts)) WITH (url='http://localhost:9200/', bulk_concurrency=1, batch_size=1048576);
CREATE INDEX idxso_users ON so_users USING zombodb (zdb('so_users', ctid), zdb(so_users)) WITH (url='http://localhost:9200/', bulk_concurrency=1, batch_size=1048576);
CREATE INDEX idxso_comments ON so_comments USING zombodb(zdb('so_comments', ctid), zdb(so_comments)) WITH (url='http://localhost:9200/', bulk_concurrency=1, batch_size=1048576);
CREATE INDEX idxwords ON words USING zombodb (zdb('words', ctid), zdb(words)) WITH (url='http://localhost:9200/', bulk_concurrency=1, batch_size=1048576);
CREATE INDEX idx_zdb_products ON products USING zombodb(zdb('products', products.ctid), zdb(products)) WITH (url='http://localhost:9200/');
